### DISCUSSION TITLE
No compile_command.json when build bear

### DISCUSSION THREAD

**Discussion by allen-c** [rizsotto/Bear#590] [Q&A]
I add set(CMAKE_EXPORT_COMPILE_COMMANDS ON) in the top CMakeLists.txt, but after building, there is no compile_commads.json file. Buiding platfrom as following:
uname -a
Linux allen-nuc 6.5.0-41-generic #41~22.04.2-Ubuntu SMP PREEMPT_DYNAMIC Mon Jun  3 11:32:55 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
Can anybody help?

**Comment by rizsotto** [comment]
Bear is using subprojects. The main project is not compiling any files, therefore the compilation database is empty. Look into the CMake file in the root folder, and modify the part which calls the subproject, which compiles the sources. (Pass that flag in there.) Then you can find the report in the subproject build directory.
